README.md

```diff
@@ -147,3 +172,35 @@ builder
         })
     .AddConsul("myApp/jsonValues", cancellationToken);
 ```
+
+### Customizing the `ConvertToConfig` strategy
+
+Sometimes you may need more control over the conversion of raw consul KV pairs into `IConfiguration` data.  In this case you can set a custom `ConvertToConfig` function:
+
+```csharp
+builder
+    .AddConsul(
+        "myApp",
+        options =>
+        {
+            options.ConvertToConfig = kvPair =>
+            {
+                var normalizedKey = kvPair.Key
+                    .Replace(_source.KeyToRemove, string.Empty)
```

Comment:
I don't think that `_source_` is avaialable in this context:
```suggestion
                    .Replace(options.KeyToRemove, string.Empty)
```

Reply:
Yeah I suppose it doesn't make sense to try and reference `KeyToRemove` here, because a client is configuring the `options` at this point, so they would know the literal value of `KeyToRemove` at this point in the code. So we can just replace this with a hardcoded string in this example.